TAGradingServer/account/account-rubric.php

```diff
@@ -555,8 +572,32 @@ function display_files($file, &$output, $part, $indent = 1) {
     
     $min_val = (intval($question['gc_max_value']) > 0) ? 0 : intval($question['gc_max_value']);
     $max_val = (intval($question['gc_max_value']) > 0) ? intval($question['gc_max_value']) : 0;
-    
-    $output .= <<<HTML
+    if($question['gc_is_extra_credit'] == true) {
+        $output .= <<<HTML
+    <tr style="background-color: #f9f9f9;">
```

Comment:
I believe I did it that since the table was made up of cells. I tried changing the row color for the cells but it seems the cell's color over wrote it. Therefore I decided to change the each cell's color directly.